message/tests.py

```diff
@@ -14,41 +18,63 @@
 TEST_BALL_GAME = 'Basketball'
 
 
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def user2():
+    return get_user_model().objects.create_user(
+        username='testuser1',
+        password='testpass1'
+    )
+
+
+@pytest.fixture
+def saved_player(user):
```

Comment:
I think that on PR #85 we moved all the fixtures to conftest.
Can we use the conftest file instead of making a new fixture here?
Maybe even adding there `saved_player2` that is written below.

Reply:
> I think that on merged PR #85 we moved all the fixtures to conftest. Can we use the conftest file instead of making a new fixture here? Maybe even adding there `saved_player2` that is written below.

You're right, I did it